frame/nomination-pools/src/lib.rs

```diff
@@ -409,6 +409,15 @@ impl<T: Config> Delegator<T> {
 		self.active_points().saturating_add(self.unbonding_points())
 	}
 
+	pub(crate) fn active_balance(&self) -> BalanceOf<T> {
+		if let Some(pool) = BondedPool::<T>::get(self.pool_id).defensive() {
+			// TODO: this function is questionable in both name and implementation.
```

Comment:
Why do you say implementation is questionable? Makes sense to me :)

Reply:
What I mentioned in DM: using PerQuintil or hardening the saturation with u256.